### DISCUSSION TITLE
How to clip multiple highlights from a webpage without clipping the whole text in that webpage

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#83] [Q&A]
[Your reply in google group discussion](https://groups.google.com/g/tiddlywiki/c/RWACKEmd9BA/m/Xe7784zzFAAJ) talks about 

`It is possible to highlight multiple selections on a page (using the highlight rule), then select all of the page's text and clip that through selection macros (that would need to written - this is not difficult), eg @Selected#1(@web), @Selected#2(@web) etc. `

![image](https://user-images.githubusercontent.com/67494083/157093817-7a7dfad4-e16e-49b0-b018-9618eff53dba.png)

Is this regarding clipping of multiple highlights from a webpage without clipping the whole text of that webpages. If yes, how to do it?

**Comment by buggyj** [comment]
this involves writing javascript macros to extract the multiple highlighted text - I think this would fail sometimes.
